### DISCUSSION TITLE
huge performance dips after unplugging monitors.

### DISCUSSION THREAD

**Discussion by KoleckOLP** [ppy/osu#18646] [Q&A] [answered]
Hello, yesterday I installed a new cpu cooler into my pc and when I plugged my monitors back in (probably in different order than before)
I am getting huge performance dips,
I was used to having around 2000 fps and now I get dips much bellow that
my draw time used to be around 0.4-0.5ms nad now it's even over 0.6ms
and in song select the framerate dips even bellow 400 fps

and what is strange is that in the setting if I try to change monitor from 0 to 1 the game window flashed but does not switch monitor, when using fullscreen.
is there a way to fix this?

**Comment by peppy** [comment, marked as answer]
You haven't provided enough information to help in any way.

If you're on a laptop with nvidia optimus, what you say makes sense and you'll need to make a choice.

Also you are expecting stupid performance that is far higher than required to play the game. Draw frames at 400 is ample.
